padertorch/contrib/examples/source_separation/pit/evaluate.py

```diff
@@ -164,9 +155,9 @@ def main(_run, batch_size, datasets, debug, experiment_dir, database_json):
 
         summary[dataset][example_id] = entry
 
-    summary_list = COMM.gather(summary, root=MASTER)
+    summary_list = mpi.COMM.gather(summary, root=mpi.MASTER)
```

Comment:
You can simply use `mpi.gather` without `COMM` in between. You can also remove `root=mpi.MASTER` because `root` defaults to `MASTER`.

Reply:
I find explicitly stating the `root` clearer since it is an example and also shows more functionalities of MPI this way.
I'll remove the redundant `COMM`.